DecipherNow/gm-ui-components
# fix: refactor LoadingSpinner

Inherit theme through the styled-components class vs the withTheme() HOC to avoid a bug in production

## To test
- Use `npm link` to link the component library to the dashboard ([see these instructions](https://github.com/DecipherNow/gm-ui-components/blob/master/CONTRIBUTING.md#npm-link))
- In the dashboard run: `npm run build && open build/index.html`
- Confirm that the app loads and these errors are no longer in the console:
![image](https://user-images.githubusercontent.com/22768918/47652551-9babc580-db5c-11e8-9859-a0234ff4d274.png)
